SNIClient.py

```diff
@@ -295,15 +326,22 @@ async def _snes_connect(ctx: SNIContext, address: str):
             return snes_socket
 
 
+class SNESRequest(typing.TypedDict):
+    Opcode: str
+    Space: str
+    Operands: typing.List[str]
+
+
 async def get_snes_devices(ctx: SNIContext) -> typing.List[str]:
     socket = await _snes_connect(ctx, ctx.snes_address)  # establish new connection to poll
-    DeviceList_Request = {
+    DeviceList_Request: SNESRequest = {
         "Opcode": "DeviceList",
-        "Space": "SNES"
+        "Space": "SNES",
+        "Operands": []  # REVIEW: Is it ok to have this here? Optional members of TypedDict is Python 3.11.
```

Comment:
I don't know whether SNI will complain if this `"Operands"` key is here. I'm hoping SNI will just ignore it.

Reply:
I\m pretty sure both SNI and QSsb2snes unpack operands and then simply ignore them for opcodes that don't use them, so besides a few extra bytes on the wire and a few extra cpu cycles it should not have any impact.

However I think leaving a note this can be "fixed" in py3.11 would be nice.